tests/unit/requirements.txt

```
@@ -48,3 +48,7 @@ passlib[argon2]
 # requirements for the proxmox modules
 proxmoxer < 2.0.0 ; python_version >= '2.7' and python_version <= '3.6'
 proxmoxer ; python_version > '3.6'
+
+#requirements for nomad_token modules
+python-nomad == 1.5.0 ; python_version <= '3.6'
+python-nomad == 2.0.0 ; python_version >= '3.7'
```

It's better to use inequalities to allow newer versions to be tested as well:
```suggestion
python-nomad < 2.0.0 ; python_version <= '3.6'
python-nomad >= 2.0.0 ; python_version >= '3.7'
```